src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -172,7 +172,7 @@ def _resolve_model(self):
 
         doc_string = doc_string.replace('\r', '').replace('\n', ' ')
         doc_string = re.sub(' +', ' ', doc_string)
-        model_name_regex = re.compile(r':return: (.*that returns )?(?P<model>[a-zA-Z]*)')
+        model_name_regex = re.compile(r':return: (?:.*?that returns (?:either )?)?(?P<model>[a-zA-Z]*)')
```

Comment:
This is dark magic.

Reply:
Good catch. Originally I added comments, it seems when I resolved conflicts manually after rebasing dev, the comments was removed accidentally. I'll add comments here to explain why this change happened.

Reply:
Could add more annotation or example? The regex is not clear.